docs/source/cmdline/delete.rst

```diff
@@ -0,0 +1,16 @@
+:program:`dandi delete`
+=======================
+
+::
+
+    dandi [<global options>] delete [<options>] [<paths> ...]
+
+Delete dandisets and assets from the server.
+
+PATH could be a local path or a URL to an asset, directory, or an entire
+dandiset.
+
+Options
+-------
+
+.. option:: --skip-missing
```

Comment:
oh -- you just manually produced them? great for now but we should look into making them generated automagically...  may be smth like https://github.com/click-contrib/sphinx-click could be used ?

Reply:
I don't really like the way the sphinx-click output looks.

Reply:
May be there is a way to improve that? I just fear that manually produced docs would quickly become our of sync with actual CLI